Arrowgene.Ddon.GameServer/Handler/ClanClanGetMyJoinRequestListHandler.cs

```diff
@@ -0,0 +1,20 @@
+using Arrowgene.Ddon.Server;
+using Arrowgene.Ddon.Shared.Entity.PacketStructure;
+using Arrowgene.Logging;
+
+namespace Arrowgene.Ddon.GameServer.Handler
+{
+    public class ClanClanGetMyJoinRequestListHandler : GameRequestPacketHandler<C2SClanClanGetMyJoinRequestListReq, S2CClanClanGetMyJoinRequestListRes>
+    {
+        private static readonly ServerLogger Logger = LogProvider.Logger<ServerLogger>(typeof(ClanClanGetMyJoinRequestListHandler));
+
+        public ClanClanGetMyJoinRequestListHandler(DdonGameServer server) : base(server)
+        {
+        }
+
+        public override S2CClanClanGetMyJoinRequestListRes Handle(GameClient client, C2SClanClanGetMyJoinRequestListReq request)
```

Comment:
Why is this empty?

Reply:
Can you put a todo comment? There was some weird packets with the mail system that as far as I could tell actually did nothing (were probably some legacy they didn't feel like removing).